mail-client/trojita/trojita-0.7.recipe

```diff
@@ -0,0 +1,134 @@
+Can be used on many platforms. One UI is not enough for everyone, but our \
+IMAP core works fine on anything from desktop computers to cell phones and \
+big ERP systems.
+Plays well with the rest of the ecosystem. We don't like reinventing \
+wheels, but when the existing wheels quite don't fit the tracks, we're \
+not afraid of making them work."
+HOMEPAGE="http://trojita.flaska.net"
+COPYRIGHT="2006-2016 Trojitá developers"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/jktjkt/trojita/archive/v$portVersion.zip"
+CHECKSUM_SHA256="e0e694aabb7dd702e1c4518dd12a17f8124d353c70a293663b4e5c6794129a29"
+ADDITIONAL_FILES="trojita.rdef"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	trojita$secondaryArchSuffix = $portVersion
+	app:Trojita$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	lib:libcrypto$secondaryArchSuffix
+	lib:libqt5$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cat
+	cmd:cmake
+	cmd:git
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+#	cmd:mimetic$secondaryArchSuffix
+#	cmd:gpgmepp$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+#	cmd:ragel
+	cmd:sed
+	cmd:python
+	cmd:sh
+	cmd:svn
+	cmd:strip
+	cmd:xargs
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	qt5${secondaryArchSuffix}_devel >= 5.5
+	devel:libssl$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+PATCH()
+{
+	sed -i 's/-pthread/-lroot/' CMakeLists.txt
+}
+
+BUILD()
+{
+	#CMake Finddir folders
+	export Qt5Core_DIR="/boot/system/data/cmake/Modules/Qt5Core"
+	export Qt5DBus_DIR="/boot/system/data/cmake/Modules/Qt5DBus"
+	export Qt5Gui_DIR="/boot/system/data/cmake/Modules/Qt5Gui"
+#	export Qt5Keychain_DIR="/boot/system/data/cmake/Modules/Qt5Keychain"
+	export Qt5LinguistTools_DIR="/boot/system/data/cmake/Modules/Qt5LinguistTools"
+	export Qt5Network_DIR="/boot/system/data/cmake/Modules/Qt5Network"
+	export Qt5Sql_DIR="/boot/system/data/cmake/Modules/Qt5Sql"
+	export Qt5Svg_DIR="/boot/system/data/cmake/Modules/Qt5Svg"
+	export Qt5Test_DIR="/boot/system/data/cmake/Modules/Qt5Test"
+	export Qt5WebKitWidgets_DIR="/boot/system/data/cmake/Modules/Qt5WebKitWidgets"
+
+	#Downloading i10n, requires network connection
+	python l10n-fetch-po-files.py
+
+	#Creating Build dir
+	mkdir -p build
+	cd build
+	cmake .. \
+		-DCMAKE_INSTALL_PREFIX:PATH=$appsDir/Trojita \
+		-DCMAKE_INSTALL_LIBDIR:PATH=$appsDir/Trojita/lib \
+		-DCMAKE_INSTALL_DATAROOTDIR:PATH=$dataDir/Trojita
+
+	#Make Trojita, but just in INSTALL(), we don't want to compile it 2 times....
+	#make $jobArgs
+}
+
+INSTALL()
+{
+	# Add Haiku resources
+	local APP_SIGNATURE="application/x-vnd.flaska.org-Trojita"
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3 | tr -d '[:alpha:]'`"
```

Comment:
> Somehow after adding resources the icon won't show up. Something wrong...

MINOR is empty because $portVersion is "0.7".
Consequently, addResourcesToBinaries fails to process the rdef.
Hardcoding MINOR should fix the issue.
```
	local MINOR="0"
```